packages/sit-onyx/src/components/TestInput/TestInput.vue

```diff
@@ -34,11 +48,22 @@ const handleChange = (event: Event) => {
 </script>
 
 <template>
-  <label>
+  <label class="input" :class="{ 'input--touched': isTouched }">
     <span class="input__label" :class="{ 'input__label--required': props.required }">
       {{ props.label }}
     </span>
-    <input v-model="value" :required="props.required" @change="handleChange" />
+    <input
+      v-model="value"
+      :required="props.required"
+      :minlength="props.minlength"
+      :maxlength="props.maxlength"
+      :min="props.min"
+      :max="props.max"
+      :type="props.type"
+      :pattern="props.pattern"
```

Comment:
```suggestion
      v-bind="props"
```

We could use this shortcut to remove some redundancies

Reply:
done :)